src/ui/SettingsButton.tsx

```diff
@@ -45,15 +45,23 @@ export default function SettingsCard({ navigator }: { navigator: Navigator }) {
           <Button onClick={open}>Settings</Button>
         </PopoverTrigger>
         <PopoverContent>
-          <PopoverHeader fontWeight="semibold">Settings</PopoverHeader>
+          <PopoverHeader fontWeight="semibold" variant="setting">
+            Settings
+          </PopoverHeader>
           <PopoverArrow />
           <PopoverCloseButton />
           <PopoverBody>
             <ToggleGroup defaultValue="publisher">
-              <ToggleButton value="publisher">Publisher</ToggleButton>
-              <ToggleButton value="serif">Serif</ToggleButton>
-              <ToggleButton value="sans-serif">Sans-Serif</ToggleButton>
-              <ToggleButton value="dyslexia-friendly">
+              <ToggleButton value="publisher" variant="setting">
+                Publisher
+              </ToggleButton>
```

Comment:
I think we are only going to have the toggle button in the settings card, so I would actually pass `variant="toggle"` into the `<Button>` you use in there, instead of passing it every time you render a `ToggleButton`.